projects/1.19/assets/create-alloyed/alloyed/lang/zh_cn.json

```
@@ -0,0 +1,64 @@
+{
+  "advancements.alloyed.bronze_ingot.description": "获取一些青铜，一种装饰性与音乐性并举的金属",
+  "advancements.alloyed.bronze_ingot.title": "另一种铜合金",
+  "advancements.alloyed.bronze_instruments.description": "合成一个青铜乐器，是时候开始作曲了！",
+  "advancements.alloyed.bronze_instruments.title": "重金属音乐",
+  "advancements.alloyed.steel_ingot.description": "获取一些钢铁，一种坚硬且结实的金属。",
+  "advancements.alloyed.steel_ingot.title": "工业铁",
+  "alloyed.ponder.bronze_bell_decoration.header": "青铜钟",
+  "alloyed.ponder.bronze_bell_decoration.text_1": "青铜钟是一种由青铜制成的装饰方块",
+  "alloyed.ponder.bronze_bell_decoration.text_2": "想要取得良好的装饰效果？试试悬挂在锁链……",
+  "alloyed.ponder.bronze_bell_decoration.text_3": "……栅栏……",
+  "alloyed.ponder.bronze_bell_decoration.text_4": "……或铁栏杆上。",
+  "alloyed.ponder.bronze_bell_instrument.header": "用青铜钟创作音乐",
+  "alloyed.ponder.bronze_bell_instrument.text_1": "当青铜钟被击打时会发出声响。",
+  "alloyed.ponder.bronze_bell_instrument.text_2": "……或者接收到红石信号时。",
+  "alloyed.ponder.bronze_bell_instrument.text_3": "扳手可调节音调。",
+  "alloyed.ponder.bronze_bell_instrument.text_4": "也可利用机械手来自动敲钟。",
```

这些ponder文本应当注意语序与衔接。为了基本保持原文语序（这玩意是要配合动画的），有时不得不大幅度更改语句结构。

```suggestion
  "alloyed.ponder.bronze_bell_instrument.text_1": "若要使青铜钟发声，可以击打它……",
  "alloyed.ponder.bronze_bell_instrument.text_2": "……或者施加红石信号。",
  "alloyed.ponder.bronze_bell_instrument.text_3": "可以用扳手调节音调。",
  "alloyed.ponder.bronze_bell_instrument.text_4": "也可利用机械手来自动敲钟。",
```